test

```diff
@@ -118,7 +118,7 @@ function cov_pass {
 	# use 30m timeout because e2e coverage takes longer 
 	# due to many tests cause etcd process to wait
 	# on leadership transfer timeout during gracefully shutdown
-	go test -tags cov -timeout 30m -v ${REPO_PATH}"/e2e"
```

Comment:
I don't think the coverprofile tag at here is necessary.  We define the coverprofile path for each e2e test at https://github.com/coreos/etcd/blob/master/e2e/etcd_spawn_cov.go.go#L44.